src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2727,7 +2727,7 @@ private string GenerateCertificateThumbprintParameter()
 
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-windows`
WSMan Config Provider tab complete tests.Tab completion gets dynamic parameters for 'localhost\plugin' using '-pr'
```powershell
Expected a collection with size 1, but got collection with size 4 @(System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult).
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1211
1211:         $res.CompletionMatches | Should -HaveCount $expected.Count
```
Tests for (error, warning, etc) action preference.action preference of Ignore cannot be set as a preference variable
```powershell
Expected an exception, with FullyQualifiedErrorId 'System.NotSupportedException,Microsoft.PowerShell.Commands.GetProcessCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 40
40:             } | Should -Throw -ErrorId 'System.NotSupportedException,Microsoft.PowerShell.Commands.GetProcessCommand' -PassThru
```